src/utils/utilsTests/itemFilterUtils.test.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  isRecapLocation,
+  combineRecapLocations,
+  parseItemFilterQueryParams,
+  buildItemFilterQueryParams,
+  buildAppliedFiltersString,
+} from "../itemFilterUtils"
+import testAggregations from "../../../__test__/fixtures/testAggregations"
+import { ItemFilterData } from "../../models/itemFilterData"
+
+describe("Item Filter Utils", () => {
+  describe("isRecapLocation", () => {
+    it("returns true for a recap location", () => {
+      expect(isRecapLocation("loc:rc2ma")).toBe(true)
+    })
+    it("returns false for a recap location", () => {
```

Comment:
Description should be for a "non-recap" location, right?

Reply:
yes!